README.md

```diff
@@ -87,6 +87,7 @@ Builds are conducted by CircleCI, and upon successful build of the `main` branch
 - `getCanaryStage` - Gets the stage of a canary deployment from an environment variable, and validates its value.
 - `getDockerTag` - Gets the Docker tag from an environment variable, and validates its value.
 - `getWatermark` - Gets the watermark from an environment variable.
+- `getTtl` - Gets the TTL datetime from an environment variable.
```

Comment:
My brain wants this to say `getTTL`  but at least your consistent with the other abbreviations